site/docs/spec.md

```diff
@@ -139,7 +139,7 @@ Partition specs capture the transform from table data to partition values. This
 | **`truncate[W]`** | Value truncated to width `W` (see below)                     | `int`, `long`, `decimal`, `string`                                                                        | Source type |
 | **`year`**        | Extract a date or timestamp year, as years from 1970         | `date`, `timestamp(tz)`                                                                                   | `int`       |
 | **`month`**       | Extract a date or timestamp month, as months from 1970-01-01 | `date`, `timestamp(tz)`                                                                                   | `int`       |
-| **`day`**         | Extract a date or timestamp day, as days from 1970-01-01     | `date`, `timestamp(tz)`                                                                                   | `int`       |
+| **`day`**         | Extract a date or timestamp day, as date                     | `date`, `timestamp(tz)`                                                                                   | `date`      |
```

Comment:
This should change the type but not description.

Reply:
Comment addressed. Please review, thanks!

Reply:
@rdblue Comment addressed, Only "Return type" is changed. Please review at your most convenience, thanks!

Reply:
@rdblue Would you please help to review this change at your most convenience?

Reply:
Hi @rdblue Would you please help to review the change at your most convenience ? The comment addressed.